PIconnect/PIAF.py

```diff
@@ -218,6 +218,13 @@ def units_of_measurement(self):
     def _current_value(self):
         return self.attribute.GetValue().Value
 
+    def _update_value(self, af_value_obj, update_mode, buffer_mode):
```

Comment:
I would prefer to call the argument just `value`, when we deprecate support for python 2 I want to add type hints to all code more explicitly.

Reply:
np